src/react/hooks/paragon/useParagonThemeUrls.test.js

```diff
@@ -0,0 +1,217 @@
+              urls: {
+                default: 'light.css',
+                brandOverride: undefined,
+              },
+            },
+          },
+        }),
+      );
+    });
+
+    it('returns expected object when configuration is valid (both Paragon + brand)', () => {
+      const config = {
+        PARAGON_THEME_URLS: {
+          core: {
+            urls: {
+              default: 'core.css',
+              brandOverride: 'brand-core.css',
+            },
+          },
+          defaults: {
+            light: 'light',
+          },
+          variants: {
+            light: {
+              urls: {
+                default: 'light.css',
+                brandOverride: 'brand-light.css',
+              },
+            },
+          },
+        },
+      };
+      mergeConfig(config);
+      const { result } = renderHook(() => useParagonThemeUrls(config));
+      expect(result.current).toEqual(
+        expect.objectContaining({
+          core: {
+            urls: {
+              default: 'core.css',
+              brandOverride: 'brand-core.css',
+            },
+          },
+          defaults: {
+            light: 'light',
+          },
+          variants: {
+            light: {
+              urls: {
+                default: 'light.css',
+                brandOverride: 'brand-light.css',
+              },
+            },
+          },
+        }),
+      );
+    });
+    it('returns expected object when core default and variants are not present, fallback to PARAGON_THEME', () => {
+      const config = {
+        PARAGON_THEME_URLS: {
+          core: {
+            urls: {
+              brandOverride: 'brand-core.css',
+            },
+          },
+          defaults: {
+            light: 'light',
+          },
+          variants: {},
+        },
+      };
+      mergeConfig(config);
+      const { result } = renderHook(() => useParagonThemeUrls());
+      expect(result.current).toEqual(
+        expect.objectContaining({
+          core: {
+            urls: {
+              default: 'localhost:8080/core.min.css',
+              brandOverride: 'brand-core.css',
+            },
+          },
+          defaults: {
+            light: 'light',
+          },
+          variants: {
+            light: {
+              urls: {
+                default: 'localhost:8080/light.min.css',
+              },
+            },
+          },
+        }),
+      );
+    });
+    it('returns expected undefined when core default and variants are not present and can not fallback to PARAGON_THEME', () => {
+      const config = {
+        PARAGON_THEME_URLS: {
+          core: {
+            urls: {
+              brandOverride: 'brand-core.css',
+            },
+          },
+          defaults: {
+            light: 'light',
+          },
+          variants: {},
+        },
+      };
+      const originalParagonTheme = global.PARAGON_THEME;
+      Object.defineProperty(global, 'PARAGON_THEME', {
+        value: 'mocked-theme',
```

Comment:
[non-blocking nit]: would it make sense to override `PARAGON_THEME` here with an `undefined` vs. string `mocked-theme`? I feel it'd be slightly more realistic (i.e., `PARAGON_THEME` not defined vs. defined with an incorrect value).